apps/web/public/landing graphic.svg

```diff
@@ -0,0 +1,472 @@
+    >
+<stop stop-color="white" />
+<stop offset="1" stop-color="#6F69A9" stop-opacity="0.8" />
+</radialGradient>
+<linearGradient
+      id="paint1_linear_1_1265"
+      x1="515.405"
+      y1="-5.47618"
+      x2="685.488"
+      y2="220.679"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#EB695A" />
+<stop offset="0.320192" stop-color="#E98462" />
+<stop offset="0.779574" stop-color="#6370B9" />
+<stop offset="1" stop-color="#A16499" />
+</linearGradient>
+<linearGradient
+      id="paint2_linear_1_1265"
+      x1="546.244"
+      y1="149.655"
+      x2="546.244"
+      y2="291.702"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#AFAFBF" />
+<stop offset="1" stop-color="#AFAFBF" stop-opacity="0" />
+</linearGradient>
+<linearGradient
+      id="paint3_linear_1_1265"
+      x1="504.19"
+      y1="175.821"
+      x2="504.19"
+      y2="205.726"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#777676" />
+<stop offset="1" stop-color="#565656" />
+</linearGradient>
+<linearGradient
+      id="paint4_linear_1_1265"
+      x1="505.125"
+      y1="140.309"
+      x2="505.125"
+      y2="205.726"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#6D6C6C" />
+<stop offset="1" stop-color="#ADADAD" />
+</linearGradient>
+<linearGradient
+      id="paint5_linear_1_1265"
+      x1="481.762"
+      y1="110.405"
+      x2="528.488"
+      y2="159"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#D34058" />
+<stop offset="1" stop-color="#6F1221" />
+</linearGradient>
+<linearGradient
+      id="paint6_linear_1_1265"
+      x1="578.952"
+      y1="231.893"
+      x2="578.952"
+      y2="254.321"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#FF90CC" />
+<stop offset="1" stop-color="#D5589B" />
+</linearGradient>
+<linearGradient
+      id="paint7_linear_1_1265"
+      x1="549.048"
+      y1="215.071"
+      x2="549.048"
+      y2="237.5"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#90EBFF" />
+<stop offset="1" stop-color="#25D4D2" />
+</linearGradient>
+<linearGradient
+      id="paint8_linear_1_1265"
+      x1="578.952"
+      y1="196.381"
+      x2="578.952"
+      y2="218.809"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#FF90CC" />
+<stop offset="1" stop-color="#D5589B" />
+</linearGradient>
+<linearGradient
+      id="paint9_linear_1_1265"
+      x1="608.857"
+      y1="215.071"
+      x2="608.857"
+      y2="237.5"
+      gradientUnits="userSpaceOnUse"
+    >
+<stop stop-color="#90EBFF" />
+<stop offset="1" stop-color="#25D4D2" />
+</linearGradient>
+<radialGradient
+      id="paint10_radial_1_1265"
+      cx="0"
+      cy="0"
+      r="1"
+      gradientUnits="userSpaceOnUse"
+      gradientTransform="translate(752.58 86.066) rotate(62.3185) scale(253.299 315.448)"
+    >
+<stop stop-color="white" />
+<stop offset="1" stop-color="#6F69A9" stop-opacity="0.8" />
+</radialGradient>
+</defs>
+</svg>
+
```

Comment:
I usually name all files with kebab-case. You should probably stick to a convention, too, but you are free to do whatever you want here.